braviarc/braviarc.py

```diff
@@ -71,8 +71,10 @@ def connect(self, pin, clientid, nickname):
 
         else:
             _LOGGER.info(json.dumps(response.json(), indent=4))
-            self._cookies = response.cookies
-            return True
+            resp = json.loads(response.content.decode('utf-8'))
```

Comment:
does response always have a content attribute snd content is alwais not null at this point?